### PR TITLE
Revert "Disable orc write by default because of https://issues.apache.org/jira/browse/ORC-1075 (#4471)"

### PR DESCRIPTION
**Pull request by wbo4958**
Depends on https://github.com/rapidsai/cudf/pull/10041 which has added the column statistic in RowIndex when writing ORC files by CUDF. I tested it locally, it can work. We need to enable GPU accelerated ORC write by default.

### CODE REVIEW

**Comment by wbo4958** [comment]
build

**Comment by tgravescs** [comment]
converted to draft since cudf pr still draft

**Review by jlowe — DISMISSED** [review]
@wbo4958 please elaborate a bit in the PR description why we're able to revert the disable commit because of the cudf changes, as this would help anyone that needs to investigate this change in the future.

File: sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuOrcFileFormat.scala [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020-2022, NVIDIA CORPORATION.
+ * Copyright (c) 2020-2021, NVIDIA CORPORATION.
```
Comment (tgravescs) [inline sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuOrcFileFormat.scala:2]: should leave 2022 in there I think

**Comment by wbo4958** [comment]
build

**Review by jlowe — APPROVED** [review]

File: sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuOrcFileFormat.scala (hunk 1 above)
Comment (wbo4958) [reply sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuOrcFileFormat.scala]: Done, Thx

**Comment by wbo4958** [comment]
build

**Pull request merged**